types/externs.d.ts

```diff
@@ -120,10 +120,14 @@ declare global {
       gatherMode?: boolean | string;
       /** Flag indicating that the browser storage should not be reset for the audit. */
       disableStorageReset?: boolean;
-      /** How emulation (useragent, device screen metrics, touch) should be applied. `none` indicates Lighthouse should leave the host browser as-is. */
-      emulatedFormFactor?: 'mobile'|'desktop'|'none';
-      /** Dangerous setting only to be used by Lighthouse team. Disables the device metrics and touch emulation that emulatedFormFactor defines. Details in emulation.js */
-      internalDisableDeviceScreenEmulation?: boolean
+
+      // TODO: formFactor should really be required, but since this type is shared in Flags, we can't reasonably do that. We'll likely need to separate SharedFlagsSettings into CLI Flags and Config Settings.
```

Comment:
~~why don't we do that today? :) I thought part of the point was to start *requiring* people to explicitly acknowledge the fact that it's mobile~~

~~Is the issue that we don't know whether to require it in the config settings or the flags?~~

I don't think I actually believe the above. I think the real explanation here is that it's unreasonable to ask for explicit setting in flags when it could be in the config. Is that right?

Reply:
yes that's right. It seems unfair to ask CLI users to use `--preset=mobile` without default.

okay technically we _could_ default the CLI to preset=mobile and that means for CLI users formFactor is always populated.  

(For DevTools, PSI, etc there's usually UI or default conventions such that requiring a formFactor choice seems OK.)

so yeah i think we could pull off a required `formFactor` if we accept CLI will have a semi-awkward default built-in.

Reply:
> so yeah i think we could pull off a required `formFactor` if we accept CLI will have a semi-awkward default built-in.

it's not that awkward :) We do it for `output`, `port`, and `hostname`, for instance

Reply:
lil chat excerpt on making `formFactor` required/nonnullable:

![image](https://user-images.githubusercontent.com/39191/101665471-dce05600-3a01-11eb-9049-f31b23ea426d.png)

When I quickly try this out, it has a few implications in core that can be handled more explicitly. But our tests is where it gets a lot busier. I think it's probably ~200 locations where we'll have to add `formFactor: mobile,` and sometimes we'll have to create and pass in a settings object when before it wasn't necessary. 

Because it's a little involved, will make for a much bigger diff, and doesn't change runtime functionality, I think we can do this in a followup. (Will chat with @brendankenny some more about this in a bit.)